adm_program/installation/db_scripts/update_3_3.xml

```diff
@@ -43,24 +43,40 @@
     <step id="380">INSERT INTO %PREFIX%_roles_rights (ror_name_intern, ror_table, ror_ror_id_parent)
                    VALUES ('category_edit', 'adm_categories', (SELECT rr.ror_id FROM %PREFIX%_roles_rights rr WHERE rr.ror_name_intern = 'category_view'))</step>
     <step id="390">alter table %PREFIX%_roles_rights add constraint %PREFIX%_FK_ROR_ROR_PARENT foreign key (ror_ror_id_parent) references %PREFIX%_roles_rights (ror_id) on delete set null on update restrict</step>
-    <step id="400" database="pgsql">ALTER INDEX idx_%PREFIX%_mem_rol_usr_id RENAME TO %PREFIX%_idx_mem_rol_usr_id</step>
-    <step id="410" database="pgsql" error="ignore">ALTER INDEX idx_%PREFIX%_msc_part_id RENAME TO %PREFIX%_idx_msc_part_id</step>
-    <step id="420" database="pgsql" error="ignore">ALTER INDEX ak_%PREFIX%_shortname RENAME TO %PREFIX%_idx_org_shortname</step>
-    <step id="430" database="pgsql">ALTER INDEX idx_%PREFIX%_prf_org_id_name RENAME TO %PREFIX%_idx_prf_org_id_name</step>
+    <step id="395" database="pgsql">CREATE INDEX IF NOT EXISTS idx_%PREFIX%_msc_part_id ON %PREFIX%_messages_content (msc_part_id)</step>
+    <step id="397" database="pgsql">CREATE INDEX IF NOT EXISTS ak_%PREFIX%_shortname ON %PREFIX%_organizations (org_shortname)</step>
+    <step id="400" database="pgsql" error="ignore">ALTER INDEX idx_%PREFIX%_mem_rol_usr_id RENAME TO %PREFIX%_idx_mem_rol_usr_id</step>
```

Comment:
Why all this `error="ignore"`? What is the problem?

Reply:
also problem of old admidio installations. There are some indices that are not in newer installations and within newer installations we got an error, but the update should not stop. Therefore ignore.